### ISSUE TITLE
Elgg 3.1 - elgg_set_ignore_access is deprecated

### ISSUE THREAD

**Issue by rohit1290** [iionly/elggx_fivestar#8]
https://github.com/iionly/elggx_fivestar/blob/0ab4d9b15cf679a1c1eeeed494f87792a61867b6/lib/functions.php#L56

Deprecated in 3.1: elgg_set_ignore_access is deprecated. Use elgg_call() with ELGG_IGNORE_ACCESS flag.

Updated Code for `elggx_fivestar_setRating` function:
```
function elggx_fivestar_setRating($entity) {

	elgg_call(ELGG_IGNORE_ACCESS, function() use ($entity) {
		$rating = elggx_fivestar_getRating($entity->guid);
		$entity->elggx_fivestar_rating = $rating['rating'];
	});

	return;
}
```

I have tested the above code.

**Comment by AbdulazizSobh** [comment]
The addition does not work on the latest version Elgg 3.3.4

**Comment by iionly** [comment]
Fixed in version 3.3.0.
